### PR TITLE
Pre-release cleanups

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: zcash_client_backend/src/data_api.rs [hunk 1]
```diff
@@ -42,6 +42,10 @@ pub mod error;
 pub mod scanning;
 pub mod wallet;
 
+/// The height of subtree roots in the Sapling note commitment tree.
+///
+/// This conforms to the structure of subtree data returned by from
```
Comment (daira) [inline zcash_client_backend/src/data_api.rs:47]:
```suggestion
/// This conforms to the structure of subtree data returned by
```

**Review by daira — DISMISSED** [review]
utACK

File: zcash_primitives/CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +41,14 @@ and this library adheres to Rust's notion of
     `orchard::builder::InProgress<orchard::builder::Unproven, orchard::builder::Unauthorized>`
     instead of `zcash_primitives::transaction::components::orchard::Unauthorized`
 - `zcash_primitives::consensus::NetworkUpgrade` now implements `PartialEq`, `Eq`
+- `zcash_primitives::legacy::OpCode` now defines a complete set of transparent opcodes,
+  instead of just a minimal set.
```
Comment (str4d) [inline zcash_primitives/CHANGELOG.md:45]: `OpCode` is a private type. The only thing that this change effects in the public API is the `Debug` impl of `Script`.

**Review by nuttycom — COMMENTED** [review]
utACK

File: zcash_primitives/CHANGELOG.md (hunk 2 above)
Comment (nuttycom) [reply zcash_primitives/CHANGELOG.md:45]: Oh, huh, I didn't realize it was private. Okay.

Comment (nuttycom) [reply zcash_primitives/CHANGELOG.md]: Amended.

**Review by str4d — APPROVED** [review]
utACK 46cc6666df6202df3bbb7188f927143f676c4d98

**Review by daira — APPROVED** [review]
reACK

**Pull request merged**